pkg/util/mount/mount.go

```diff
@@ -131,7 +131,7 @@ var _ Interface = &Mounter{}
 var _ HostUtils = &hostUtil{}
 
 // This represents a single line in /proc/mounts or /etc/fstab.
-type MountPoint struct {
+type Point struct {
```

Comment:
I'm not sure that `Point` is a full descriptor of the struct we're referring to here. A `MountPoint` is very different from a `Point` in space or otherwise.

Reply:
The argument made [here](https://github.com/kubernetes/utils/pull/100#issuecomment-511899873) is that you always refer to a Point in the context of the mount pkg, so it will always be referred to as `mount.Point`